jscomp/syntax/native_ast_derive_abstract.ml

```diff
@@ -7,9 +7,16 @@ let deprecated name =
     ("use " ^ name ^ "Get instead or use {abstract = light} explicitly")
 
 let strip_option arg_name =
+#if OCAML_VERSION =~ ">4.03.0" then
+   match arg_name with 
+   | Asttypes.Nolabel -> assert false
+   | Optional s
```

Comment:
we no longer maintain 4.02.3 for new stuff, so you don't need conditional compilation in the coming diffs